### ISSUE TITLE
Audio download 

### ISSUE THREAD

**Issue by alidan** [MrS0m30n3/youtube-dl-gui#269]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer **honestly**
- Put an `x` into all the boxes [ ] relevant to your issue (like that [x])
- Use *Preview* tab to see how your issue will actually look like

### WARNING
All invalid issues will be rejected!!

### What is the purpose of your *issue*?

- [ ] Bug report
- [x] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

Please remove any sections between (---) if they are not related to your issue

### Feature request (request for a new functionality)

Please make sure that the requested feature is **NOT** already in the [TODO](https://github.com/MrS0m30n3/youtube-dl-gui/blob/master/TODO) list

- [x] I've **verified** and **i assure** that my requested feature is **NOT** in the TODO list

#### What operating system do you use ?

---

Sorry if this isn't how you wanted the post, not quite sure what I was suppose to do 
I am on .4
I think that's the most recent version
I believe that the program if I leave it default will just download the best files it can
In the options there is something I can check that will "extract audio from video file"

Now, if what I said is correct, there appears to be no way without going into the options to just download the audio or just get the best version. 
Personally, I use this to grab videos I want to watch, however there are also videos where I just want the audio
Would it be possible to either add a check box to the programs main ui that is "I just want the best audio" or in the drop down "best audio"
It would be nice to have a toggle on the main ui instead of downloading 25 file list, leave it alone, only to find out that in the menu I still had 'extract audio from video file' checked.

**Comment by jult** [comment]
Yes, +1. Just an option "highest available quality audio/video" or "original audio as uploaded" or lossless to "WAV" would be nice to see. I tried this using the command-line version, but it is way too complex for the purpose.
Who *doesn't* want the highest quality video and audio by the way? That should be the optional part. Default should be to mimic the originally YT uploader source as much as possible. If you'd want lower than that, you would not be using youtube-dl, you'd be using one of those stupid browser add-ons.

**Comment by alidan** [comment]
just a reason for 'why not the highest quality version' I have a 4k monitor, I get 4k video from youtube, however unless you record on something that has a VERY good auto focus or you manually focus it, the image quality is never what you expect 4k to be, in these cases I would rather have a 1080p version. 

or in the case of lgr, when he takes video of a monitor, in 4k the monitor is harder to see then 1080, 1080 blurs it a bit making screens easier to see, even if the overall quality is technically lower. 

that said, a drop down for resolution limiting would also be nice, lets say i sat 1080p is what I want, if the video is available in 1440p or higher it will only gram 1080 and if its 720p or lower it will grab whatevers higher

For audio, even if it's all flac, the file size is small enough to not be an issue so whatevers best, but video... that's another story altogether.

**Comment by jult** [comment]
Complete YouTube Saver add-on had exactly that; A dropdown for all the options, with the (estimated) size of the download, and you could set your preferred default formats/sizes/resolutions in the Options. Sadly, firefox quantum came along and ruined CYS.
